static/app/main.tsx

```diff
@@ -20,7 +20,14 @@ function renderRouter(props: any) {
   );
 }
 
-const queryClient = new QueryClient();
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      // Prevents queries from being refetched if the data is < 20 seconds old
+      staleTime: 20_000,
```

Comment:
TBD if this is the value we want. We do want to override the default here (which is 0) to dedupe requests and not have to disable `refetchOnMount` everywhere.

Reply:
that's great, but I don't know what the ideal value would be here... why did you choose 20_000? was it a random value?

Reply:
I have no reason for choosing that value 🙃 

After some thinking I actually pushed a new value, 5 seconds. But that too doesn't haven't much logic behind it. I think that the default here should be something we revisit once we have more hooks using react-query so we can see what value makes most sense. And perhaps we can export some constants for when people want to increase or decrease the cache time?

Reply:
After further reviews, I changed it again... back to the default of `0`. We can revisit in the future, but users can easily override with `useQuery([...], { staleTime: 1234 })`

Reply:
I was afraid of having a default number here and I am happy that we decided to not set it